### PR TITLE
Cu 94ttnh desktop navigation bar

### PR DESCRIPTION
**Pull request by Hamza-A**
## Description
This PR aims to provide a new `<DesktopNavigationBar />` to our app only visible on desktop computers

- I created the `<DesktopNavigationBar />` and its story
- I implemented it in the `<Layout />` and refactored it accordingly
- I made sure that `<Container />` and `<DesktopNavigationBar />` stretches to 100% inside of their `<div />`
- Therefore, I created new `<div />` in `<Layout />`: the `<DesktopNavigationBarWrapper />` which stretches to 30% and the `<ChildrenContainer />` to 60% (I left 10% on purpose to aerate)
- The `<Header />` is no more displayed on desktop, it is now within the `<DesktopNavigationBar />`
- To manage responsiveness, I preferred using `display: none; / display: block;` rather than `visibility: hidden; / visibility: visible;` because I wanted the component to be completely out of the DOM, I implemented this change in `MobileNavigationBar />` and `<DesktopBackground />`

## Type of change

- [ ] **Chore** (non-breaking change which refactors / improves the existing code base).
- [ ] **Bug fix** (non-breaking change which fixes an issue).
- [x] **New feature** (non-breaking change which adds functionality).
- [ ] **Breaking change** (fix or feature that would cause existing functionality to not work as expected).

## How Has This Been Tested?
I added a e2e testing file for this new component

## Screenshots

image

## Checklist:

- [x] My code follows the style [**contributing guidelines**][contributing_file] of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.

[contributing_file]: https://github.com/fewlinesco/connect-account/blob/master/README.adoc

### CODE REVIEW

File: src/components/display/fewlines/BlackSwitchIcon/BlackSwitchIcon.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+import { BlackSwitchIcon } from "./BlackSwitchIcon";
+
+export default {
+  title: "components/BlackSwitchIcon",
```
Comment (Fuzznimp) [inline src/components/display/fewlines/BlackSwitchIcon/BlackSwitchIcon.stories.tsx:6]:
```suggestion
  title: "icons/BlackSwitchIcon",
```

Comment (Hamza-A) [reply src/components/display/fewlines/BlackSwitchIcon/BlackSwitchIcon.stories.tsx]: OK fixed ea870e5, in addition I added a `<Wrapper />` over all page's stories to make them displayed as on the application (`width: 60%` for desktop `width: 90%` for mobile)

Comment (Hamza-A) [reply src/components/display/fewlines/BlackSwitchIcon/BlackSwitchIcon.stories.tsx]: Besides, I created the `/icons` folder 135920b

Comment (Hamza-A) [reply src/components/display/fewlines/BlackSwitchIcon/BlackSwitchIcon.stories.tsx]: I created the `<StoriesContainer />` c4b0afb

File: src/components/display/fewlines/Account/Account.tsx [hunk 2]
```diff
@@ -5,9 +5,9 @@ import styled from "styled-components";
 import { Container } from "../Container";
 import { H1 } from "../H1/H1";
 import { H2 } from "../H2/H2";
-import { LoginsIcon } from "../LoginsIcon/LoginsIcon";
+import { LoginsIcon } from "../Icons/LoginsIcon/LoginsIcon";
+import RightChevron from "../Icons/RightChevron/RightChevron";
```
[the remark this answers is not in the document]
Comment (Hamza-A) [reply src/components/display/fewlines/Account/Account.tsx]: OK fixed 6d9def9 there was an extra `export default` for `<RightChevron />` that I removed

File: src/components/display/fewlines/Logins/Logins.stories.tsx [hunk 3]
```diff
@@ -55,5 +57,18 @@ export const LoginsPage = (): JSX.Element => {
       },
     ],
   };
-  return <Logins sortedIdentities={mockedSortedResponse} />;
+  return (
+    <Wrapper>
+      <Logins sortedIdentities={mockedSortedResponse} />
+    </Wrapper>
+  );
```
Comment (Fuzznimp) [inline src/components/display/fewlines/Logins/Logins.stories.tsx:64]: Use story container.

File: tests/e2e/desktop-navigation-bar.test.ts [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  beforeAll(async () => {
+    await openBrowser({ args: ["--window-size=1440,1000"], headless: true });
+  });
+
+  afterAll(async () => {
+    await closeBrowser();
+  });
+
+  test("Log in with a desktop computer, use the navbar to go to the logins page and go back to home page", async (done) => {
+    try {
```
Comment (Fuzznimp) [inline tests/e2e/desktop-navigation-bar.test.ts:27]: Add `expect.assertions(6)` before the `try/catch`.

File: src/components/display/fewlines/Logins/Logins.stories.tsx (hunk 3 above)
Comment (Hamza-A) [reply src/components/display/fewlines/Logins/Logins.stories.tsx]: OK fixed a32423e

File: tests/e2e/desktop-navigation-bar.test.ts (hunk 4 above)
Comment (Hamza-A) [reply tests/e2e/desktop-navigation-bar.test.ts:29]: OK fixed a32423e

**Review by Fuzznimp — APPROVED** [review]

**Pull request merged**